docs/other-tools/kubernetes/kind.md

```diff
@@ -49,9 +52,9 @@ chmod +x /usr/bin/kubectl
 
 - Test to ensure the version you installed is up-to-date:
```

Comment:
how does the code below tell us if we are up to date, it will tell us what version we are on but how do we know what the current version is/should be?

Reply:
I update it to just test/verify the `kubectl` is installed.